wiki/Twitter/pl.md

```diff
@@ -1,17 +1,24 @@
----
-outdated_translation: true
----
-
 # Twitter
 
-## My także tweetujemy!
+osu! posiada kilka kont na Twitterze; każde z nich pełni inne funkcje. Pomimo niewielkiej aktywności oferują one wystarczające informacje o tym, co dzieje się w świecie osu!.
+
+## Serwisy
+
+| Awatar | Nazwa | Opis |
+| :-: | :-: | :-- |
+| ![osu!status avatar](img/osustatus.jpg) | [@osustatus](https://twitter.com/osustatus) | Informacje dotyczące problemów ze stroną lub [Bancho](/wiki/Bancho_(server)). |
+| ![osu! support avatar](img/osusupport.jpg) | [@osusupport](https://twitter.com/osusupport) | Pomoc w sprawach związanych z kontem oraz społecznością osu!. Działa dzięki [drużynie do pomocy w sprawach związanych z kontami](/wiki/People/The_Team/Account_support_team). |
+
+## Społeczność
+
+| Awatar | Nazwa | Opis |
+| :-: | :-: | :-- |
+| ![osu! avatar](img/osugame.jpg) | [@osugame](https://twitter.com/osugame) | Oficjalne źródło newsów i ogłoszeń. |
+| ![BanchoBoat avatar](img/banchoboat.jpg) | [@banchoboat](https://twitter.com/banchoboat) | Humorystyczne rozluźnienie sytuacji, gdy coś pójdzie nie tak. |
+| ![NAT avatar](img/osu_nat.png) | [@osu_nat](https://twitter.com/osu_nat) | Newsy, ogłoszenia, krótkie ankiety społecznościowe prowadzone przez [NAT](/wiki/People/The_Team/Nomination_Assessment_Team) (nieużywane przez [główny team osu!](/wiki/People/The_Team)). |
```

Comment:
*not run* is translated a little inaccurately here to me. I'd say that the intent of the original is closer to

```suggestion
| ![NAT avatar](img/osu_nat.png) | [@osu_nat](https://twitter.com/osu_nat) | Newsy, ogłoszenia, krótkie ankiety społecznościowe prowadzone przez [NAT](/wiki/People/The_Team/Nomination_Assessment_Team) (konto nie należy do [głównego zespołu osu!](/wiki/People/The_Team)). |
```

Suggestion also removes anglicised vocabulary (*team* snuck past there).